From 6ba1150fa6c460937fcfb96a6b4cf166b85f865c Mon Sep 17 00:00:00 2001
From: Piotr Gorski <lucjan.lucjanov@gmail.com>
Date: Tue, 28 Jun 2022 11:36:19 +0200
Subject: [PATCH 36/36] Revert "Revert "lrng-5.18: drop
 EXPORT_SYMBOL_GPL(add_bootloader_randomness)""

This reverts commit 4880c03a5eeefacbe6134b53e241819894ca19d3.

Signed-off-by: Piotr Gorski <lucjan.lucjanov@gmail.com>
---
 drivers/char/lrng/lrng_interface_random_kernel.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/char/lrng/lrng_interface_random_kernel.c b/drivers/char/lrng/lrng_interface_random_kernel.c
index 81060ba98..ca2efefb1 100644
--- a/drivers/char/lrng/lrng_interface_random_kernel.c
+++ b/drivers/char/lrng/lrng_interface_random_kernel.c
@@ -108,7 +108,6 @@ void add_bootloader_randomness(const void *buf, size_t size)
 			     IS_ENABLED(CONFIG_RANDOM_TRUST_BOOTLOADER) ?
 			     size * 8 : 0);
 }
-EXPORT_SYMBOL_GPL(add_bootloader_randomness);
 
 /*
  * Callback for HID layer -- use the HID event values to stir the entropy pool
-- 
2.37.0.rc0.15.g3b9a5a33c2

